Examples/Nodify.Playground/SettingsView.xaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+
+namespace Nodify.Playground
+{
+    public partial class SettingsView : UserControl
+    {
+        public static readonly DependencyProperty SettingsItemsProperty =
+            DependencyProperty.Register("SettingsItems", typeof(object), typeof(SettingsView),
```

Comment:
`Items` or `ItemsSource` would be a better name. And the property should be type-safe. 

```csharp
public static readonly DependencyProperty ItemsProperty =
    DependencyProperty.Register(nameof(Items), typeof(IEnumerable<ISettingViewModel>), typeof(SettingsView));

public IEnumerable<ISettingViewModel> Items
{
    get => (IEnumerable<ISettingViewModel>)GetValue(ItemsProperty);
    set => SetValue(ItemsProperty, value);
}
```

Reply:
Please use `nameof(Items)` instead of string literal when you register the property.

Also use expression body for `get`, `set` accessors if they are one line.

`new PropertyMetadata(null)` is not necessary